util/lib_posix.cc

```diff
@@ -0,0 +1,614 @@
+//  Copyright (c) 2013, Facebook, Inc.  All rights reserved.
```

Comment:
do you think posix_io.cc and posix_io.h (or posix_file?) a better name? It will catch the the I/O aspects or file of the file.